src/ATA.Bluebook/ATA.Bluebook.Web/wwwroot/css/site.css

```diff
@@ -1,11 +1,24 @@
 .login-page {
```

Comment:
Add border-bottom to the active class for large screen

Reply:
Done